de.json

```diff
@@ -831,18 +831,18 @@
         "Settings.Dash.Curvature" : "Krümmung",
         "Settings.Dash.OpenCloseSpeed" : "Öffnungs/Schließ-Geschwindigkeit",
 
-        "Settings.LeapMotion.Name" : "Leap Motion",
+        "Settings.LeapMotion.Name" : "Leap Motion Tracker",
```

Comment:
Not sure about this one tbh, the English one for that JSON key is also just "Leap Motion".